wasm/js/animation_callback_handler.js

```diff
@@ -41,6 +76,18 @@ function AnimationCallbackHandler() {
         });
 
         this.onAfterCallbacks();
+
+        if (_fpsCounter) {
+            _fpsCounter.frameComplete();
+        }
+    }
+
+    this.enableFPSCounter = function(enable) {
```

Comment:
Another idea to add in the ^^ `options` is `fpsEventName` where the consumer passes along an event name. Then instead of (or alongside of) setting the innerhtml with the FPS number, we dispatch a custom event with the event name passed in. The consumer would have an event listener that allows them to deal with the fps number however they want. This would be useful for example where instead of setting HTML for this value, they would rather have it console logged, or sent to a logging tool, etc.

Reply:
Let's just do this idea, actually.

* User provides a callback that receives an FPS callback once/second.
* If no callback is provided, we add a simple div to the page and dump FPS.

Reply:
Makes sense to me

Reply:
Done!